README.md

```diff
@@ -2,3 +2,31 @@
 a bot to help assist the crates.io, website, and other rust ops teams
 
 [![Build Status](https://travis-ci.com/rust-lang/ops-bot.svg?token=hHeDp9pQmz9kvsgRNVHy&branch=master)](https://travis-ci.com/rust-lang/ops-bot)
+
+**This project is currently under development**
+
+## Intro
+
+This is a chat bot for [Discord](https://discordapp.com/) to manage the infrastructure of [crates.io](https://crates.io/).
+
+Most of the infrastructure for crates.io is hosted by [Heroku](https://www.heroku.com/). Currently if someone
+is to manage any part of the infrastructure for crates.io, they must have full credentials to Heroku. 
+
+The purpose of this bot is to allow basic infrastructure management tasks to be conducted through the Rust Language Discord server.
+It would be limited to specific members of the crates.io team, but it would allow more people to assist with basic infrastructure
+management without needing full credentials to Heroku.
+
+### Stage 1
+
+Stage 1 of this bot will allow an authorized Discord member to:
+* Update select environmental variables (not including things like API keys)
+* Restart the application (which fixes most issues)
+* Scale the application
+* Change Dyno types
+* Deploy the application
+* Rollback a deployment
+* Block/unblock ips
```

Comment:
This is currently done via environment variables, so this is actually already covered by the first bullet point.

Reply:
I think it'd be nice to have a dedicated command for blocking/unblocking IPs though; especially because the value of that environment variable is a comma-separated list of like 20 IP addresses and it's currently pretty hard to read and work with. For example, I'd rather be able to tell the chatbot:

```
block IP 127.0.0.1 
unblock IP 127.0.0.1
```

instead of:

```
read env var BLOCKED_IPS
> 192.168.0.1,192.168.0.2,192.168.0.3,192.168.0.4,192.168.0.5,192.168.0.6,192.168.0.7,
write env var BLOCKED_IPS= 192.168.0.1,192.168.0.2,192.168.0.3,192.168.0.4,192.168.0.5,192.168.0.6,192.168.0.7,127.0.0.1
```

etc.

So I like that this is a separate item :)

Reply:
Note that one of @sgrif's requirements in #4 was to prevent getting the full list of blocked IPs from the bot.

Reply:
I can see that having this as a separate feature is more convenient.

Blocking access to the full list of blocked IPs seems overly restrictive to me, though. We will have to give anyone helping with ops in any meaningful way full access to the logs anyway. The list of blocked IP addresses feels less sensitive than that.

Anyway, this is getting much too detailed for now – we will need access control for environment variables in any case, so there is no need to decide on specific ones now.